### PR TITLE
Add option to fetch login or email

### PR DESCRIPTION
**Pull request by mzellho**
This PR adds a little option to lookup the login for a credential, similar to the password-option.

### CODE REVIEW

**Comment by Mikescops** [comment]
Hello,

Thanks for taking the time to open a PR here. I like the idea but I think `dcli login` is a little bit confusing as it may let think that's the way to login into the Dashlane account. 
Also, to avoid duplicating much code I think it would be better to use an option on `dcli password` instead. That way we can reuse most of the code and just make a condition when doing the copy in the pastebin. Something like `dcli password --login`.

Let me know what you think about this.

**Comment by mzellho** [comment]
My pleasure, thanks so much for this awesome tool that saves me so much time every day.

Initially, I was going with "username", but then realised that what I wanted to fetch is the login. So if you are okay with "user" we could also use this?

Of course, sub-options of password would also be possible and produce less duplicate code.

Your call, really 🙂

**Comment by Mikescops** [comment]
Alright, I'd go for the sub-option then, when I initially made the different commands I followed the content-types that we have in the vault : passwords, secure notes, secrets.

**Comment by mzellho** [comment]
Alright, I added the sub-options discussed and also introduced to fetch the email. Any other bits of the credentials worth fetching via the cli?

File: src/commands/index.ts [hunk 1]
```diff
@@ -52,9 +52,17 @@ export const rootCommands = (params: { program: Command }) => {
                 '-o, --output <type>',
                 'How to print the passwords. The JSON option outputs all the matching credentials'
             )
-                .choices(['clipboard', 'password', 'json'])
+                .choices(['clipboard', 'console', 'json'])
                 .default('clipboard')
         )
+        .addOption(
+            new Option(
+                '-c, --credential <type>',
```
Comment (Mikescops) [inline src/commands/index.ts:60]: Nitpick: Can you name this `field` please?

**Review by Mikescops — CHANGES_REQUESTED** [review]
Looks good to me, just a naming recommendation and we can go with this, thanks!

**Comment by mzellho** [comment]
@Mikescops done, thank you!

**Review by Mikescops — APPROVED** [review]
Tested locally and it works well.

**Pull request merged**